Composer/packages/ui-plugins/virtual-assistant-creation/src/shared/types.ts

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
You don't need to create a shared folder, consider putting types and utils directly under src/